app/controllers/organisations_controller.rb

```diff
@@ -69,10 +69,12 @@ def create
        redirect_to organisations_path and return false
      end
     @organisation = Organisation.new(org_params)
+    @categories_start_with = Category.first_category_name_in_each_type
 
     if @organisation.save
       redirect_to @organisation, notice: 'Organisation was successfully created.'
     else
+      flash[:error] = @organisation.errors.full_messages.join("<br/>").html_safe
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)